projects/tutorials/minigrid_tutorial_conds.py

```diff
@@ -0,0 +1,171 @@
+from typing import Dict, Optional, List, Any, cast
```

Comment:
Would be great to make this into a full tutorial at some point.

Reply:
Well, this is an unrealistic policy design (I just used it for development/debugging). I think it's ok-ish as a test, but I'll think about a more interesting design for a tutorial 👍